docs/monitoring/custom-metrics.md

```diff
@@ -1,6 +1,13 @@
 title: Custom Metrics
 description: Sematext infrastructure and application monitoring service exposes APIs and provides libraries that let you send custom metrics and turn them into dashboards and real-time visualizations
 
+
+## Deprecated!
+
+Needs revamp after we implement chart builder and new custom metrics.
```

Comment:
Should this be in comments/hidden?

Reply:
Something for @adnanrahic . It seems it is showing here because I sent PR to merge into master, instead of into develop branch, will adjust that.